_posts/2017-09-08-web.md

```diff
@@ -12,9 +12,84 @@ hierarchy:
 ---
 
 
-## Web apps
+## An Introduction 
 
-TODO
+Welcome to the StartHacking web track! The goal of section is to provide some background information on how the Internet works, as well as provide the conceptual and technical skills needed to provide a successful career in web development.
+
+
+#### How does the Internet work? 
+
+The Internet has become the backbone of our modern-day world, yet few individuals take the time to understand what it is, and how it works. Since it has become such a large part of our lives, a good understanding is needed to become a great Web Developer, and use this tool most effectively.
+
+This section will explain the underlying infrastructure and technologies that make the Internet work; it won't provide a very detailed explanation, and resources will be provided at the end for those who wish to learn more. Any comments, questions, suggestions, etc are welcome and may be directed to the author(s) at [team@hackmit.org](mailto:team@hackmit.org).
```

Comment:
> Any comments, questions, suggestions, etc are welcome and may be directed to the author(s) at team@hackmit.org

Instead of telling them to email us, let's link them to the [contributing](https://starthacking.org/contributing.html) page, which has instructions on how to make a GitHub issue about content (which is how we want to manage the content curation process).